### PR TITLE
Finetuned Chinese version of installation.md

### PR DESCRIPTION
**Pull request by ryuuTAylor**
* doc: N/A

* fix: Fintuned the Chinese Version

* doc: remove unused information

# Preamble

For Pull Request initiator, please read the following steps carefully. 请合并请求的发起人仔细阅读下列步骤。

---

If you are creating a `Pull Request` about listed topic below, please go the the `Preview` tab and click the relative link:
如果您要创建一个有关下列内容的的合并请求，请切换到 `Preview` 面板并点击下方相关的链接:

* Select [Awesome-PETs](?expand=1&template=pull_request_template_pets.md) for files under docs/awesome-pets

---

Select a proper `Pull Request` template can accelarate the review process, thanks for your cooperation.
选择一个正确的`合并请求`模版可以加速审查流程，谢谢您的配合

If your `Pull Request` topic is not amoung the list above, please just delete all prepopulated template content.
如果您的`合并请求`内容不在上述的列表里，请删除提前填入的模版内容。

---

Thanks for your contribution to SecretFlow project
感谢您对于 SecretFlow 项目的贡献

### CODE REVIEW

**Comment by ryuuTAylor** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/locales/zh_CN/LC_MESSAGES/getting_started/installation.po [hunk 1]
```diff
@@ -24,31 +24,31 @@ msgstr "安装"
 msgid ""
 "The simplest way to try SecretFlow is to use [offical docker "
 "image](#option-2-from-docker) which ships with SecretFlow binary."
-msgstr "最简单的尝试SecretFlow的方式是使用带有SecretFlow的 **官方 docker image** 。"
```
Comment (zhangxingmeng) [inline docs/locales/zh_CN/LC_MESSAGES/getting_started/installation.po:27]: 这里没必要将SecretFlow改为隐语。

File: docs/locales/zh_CN/LC_MESSAGES/getting_started/installation.po [hunk 2]
```diff
@@ -72,31 +72,31 @@ msgstr "CPU/Memory: 推荐最低配置是 8C16G."
...
 #: ../../getting_started/installation.md:24
 msgid "Option 1: from pypi"
-msgstr "方式 1: 从pypi"
+msgstr "方式1: 通过pypi安装"
 
 #: ../../getting_started/installation.md:25
 msgid ""
 "For users who want to try SecretFlow, you can install [the current "
 "release](https://pypi.org/project/secretflow/)."
-msgstr "请安装 `当前release <https://pypi.org/project/secretflow/>`__ 。"
```
Comment (zhangxingmeng) [inline docs/locales/zh_CN/LC_MESSAGES/getting_started/installation.po:81]: 这里的release是有特殊意义的，不建议修改

File: docs/locales/zh_CN/LC_MESSAGES/getting_started/installation.po [hunk 3]
```diff
@@ -324,15 +324,15 @@ msgid ""
 "For more information, please visit [the GPU docker images at "
 "Dockerhub](https://hub.docker.com/r/secretflow/secretflow-gpu)."
 msgstr ""
-"对于更多的信息，请访问 `Dockerhub上的 GPU docker 镜像 <https://hub.docker.com/r/secretflow/secretflow-gpu>`__"
+"对于更多的信息，请访问 `Dockerhub上的GPU docker镜像 <https://hub.docker.com/r/secretflow/secretflow-gpu>`__"
```
Comment (zhangxingmeng) [inline docs/locales/zh_CN/LC_MESSAGES/getting_started/installation.po:327]: 请在英文和中文之间加一个空格，像之前版本一样

**Comment by zhangxingmeng** [comment]
Thank you so much for your contribution. However, we have noticed some issues with the accuracy of the content in this PR, and therefore we are unable to merge it at the moment. We highly appreciate your involvement in the [Secre](https://github.com/secretflow/secretflow/pull/648#)[tFlow](https://github.com/secretflow/secretflow/pull/648#) Open Source Community, especially with regards to [OSCP](https://github.com/orgs/secretflow/projects/1/views/1). Once again, thank you for your contribution.

**Comment by zhouaihui** [comment]
You can reopen this PR after making appropriate modifications.

**Pull request closed without merging**